### PR TITLE
The Restaurant App

### PR DESCRIPTION
**Pull request by acushlakoncept**
This PR contains the full implementation of a Simple Restuarant Page App using webpack. DOM manipulation was also implemented to dynamically render the various pages: Home, Menu, and Contact.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hi @acushlakoncept  👋

**You've done quite well with this project.👍**

However, let's make some changes and we would be good to go.

### **Please refer to the comments made on your code below**

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+          <a class="nav-link active" href="#">Home</a>
+        </li>
+        <li class="nav-item pt-4">
+          <a class="nav-link" href="#">Menu</a>
+        </li>
+        <li class="nav-item pt-4">
+          <a class="nav-link" href="#">Contact</a>
+        </li>
+      </ul>
+    </div>
```
Comment (mikenath223) [inline dist/index.html]: - [x] You are expected to make index.html very lean. Please remove this nav element and dynamically create it with JavaScript instead.

File: dist/index.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    <script
+      src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js"
+      integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q"
+      crossorigin="anonymous"
+    ></script>
+    <script
+      src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
+      integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
+      crossorigin="anonymous"
+    ></script>
```
Comment (mikenath223) [inline dist/index.html]: - [x] Also please remove this script and its contents and refer to [this link](https://getbootstrap.com/docs/4.0/getting-started/webpack/) for instructions on how to use bootstrap with webpack.

File: package.json [hunk 3]
```diff
@@ -17,6 +17,9 @@
   },
   "homepage": "https://github.com/acushlakoncept/restaurant-page#readme",
   "devDependencies": {
+    "eslint": "^6.8.0",
```
Comment (mikenath223) [inline package.json:24]: - [x] Please add necessary run common commands like build, development and start in package.json inside `scripts'.

**Review by Dandush03 — APPROVED** [review]
# Status: :rocket: Approved :rocket: :smile: 
# Keep it up
# :computer: Happy Coding :computer:

**Pull request merged**